specs/integration/search/CreateDatasetWidget.js

```diff
@@ -0,0 +1,12 @@
+contributeLink: 'a[title="Contribute!"]',
+publishDatasetLink: ".actions-list a:first-child",
+nextLink: "button.pointer:last-child",
+titleField: "input#title",
+descriptionField: "textarea#description",
+setFrequencyOption: 'select#frequency option[value="punctual"]',
```

Comment:
I guess you've encountered this issue where Selenium complains it wants `"` and not `'` for a valid CSS selector  ;)  That's why I'd advise using only single quotes throughout integration tests, to form a habit and avoid switching delimiters on a case-by-case basis.